GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/ListBucketsOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    /// <summary>
+    /// Options for <c>ListBuckets</c> operations.
+    /// </summary>
+    public class ListBucketsOptions
+    {
+        /// <summary>
+        /// The prefix to match. Only buckets starting with this string will be returned.
+        /// </summary>
+        public string Prefix { get; set; }
+
+        /// <summary>
+        /// The number of results to return per page.
+        /// </summary>
+        public int? PageSize { get; set; }
+    }
+
+    /// <summary>
+    /// Extension methods on <see cref="ListBucketsOptions"/> to perform defaulting,
+    /// handling either a null options reference *or* a null property value by replacing
+    /// it with the default.
+    /// </summary>
+    internal static class ListBucketsOptionsExtensions
+    {
+        internal static string GetPrefix(this ListBucketsOptions options)
+        {
+            return options?.Prefix;
+        }
+
+        internal static int GetPageSize(this ListBucketsOptions options)
+        {
+            return options?.PageSize ?? 10;
```

Comment:
Is this 10 the same default as the API? Why not let it unspecified so the API does it's own thing?